app/frontend/Shared/ContextMenu.ts

```diff
@@ -273,7 +275,9 @@ export async function openBrowser(url: URLString) {
   await appGlobal.remoteApp.openExternalURL(url);
 }
 
-export function download(url: URLString, window: any, howSaveAsDialog: boolean, filename?: string) {
```

Comment:
Why is this `export` ? Nobody else uses this. I thought this was the generic function, but it's actually used only in this file.

Please remove the `export` from all functions in this file that are not used by other files.